SplitIt/app/src/main/java/project/cs495/splitit/MainActivity.java

```diff
@@ -280,6 +280,8 @@ private String parceScanResults(ScanResults brScanResults, String groupId) {
                 .setVendor(brScanResults.merchantName().value() == null ? "Unknown" : brScanResults.merchantName().value())
                 .setDatePurchased(brScanResults.receiptDate().value())
                 .setPrice(brScanResults.total().value())
+                .setSubtotal(brScanResults.subtotal().value())
+                .setTax(brScanResults.taxes().value())
```

Comment:
Get this error when adding a new receipt: "java.lang.NullPointerException: Attempt to invoke virtual method 'float com.microblink.FloatType.value()' on a null object reference
at project.cs495.splitit.MainActivity.parceScanResults(MainActivity.java:284)
at project.cs495.splitit.MainActivity.onActivityResult(MainActivity.java:268)"